corio.py

```diff
@@ -261,6 +271,8 @@ def main(options):
 if __name__ == "__main__":
     log_cleanup()
     opts = parse_args()
+    os.environ["options"] = str(opts)
```

Comment:
Why do we need to set this in the environment? We can use the opts as however we like in the code.

Reply:
Yes, We can have another approach as well. passing options to method as parameter
activate_degraded_mode(options)